Emma/emma_libs/memoryManager.py

```diff
@@ -208,87 +198,116 @@ def createStandardReports():
+                :param colour: [str] Text colour (see HTML reference)
+                :return: None
+                """
                 yAxe = y
-                lastSectionPosition = 5
+                lastSectionPosition = 5             # Determine the deepest (with highest y value) section element, used to plot objects so they do not cross section elements; in pixel
                 biggestEndAddress = 0
                 plottedElements = []
                 distanceBetweenElements = 15
-                for index, element in enumerate(elementstoPlot):
-                    xAxeRectStart = element[Element.AddressStart] - startPoint
-                    rectLength = element[Element.AddressLength] - 1
-                    originalStartAddress = element[Element.AddressStart]
+                for index, element in enumerate(elementsToPlot):
+                    xAxeRectStart = element[Element.addressStart] - startPoint
+                    rectLength = element[Element.addressLength] - 1
+                    originalStartAddress = element[Element.addressStart]
                     fontColour = "black"
-                    if len(str(element[Element.AddressEnd])) > 11 or len(str(originalStartAddress)) > 11:          # check if address start / end fits in the rectangle
-                        yAxe = yAxe + len(str(element[Element.AddressEnd])) - 10
-                    if element[Element.AddressStart] < startPoint:
+                    if len(str(element[Element.addressEnd])) > 11 or len(str(originalStartAddress)) > 11:          # Check if address start / end fits in the rectangle, 11 (pixels) is the size of the rectangle
+                        yAxe = yAxe + len(str(element[Element.addressEnd])) - 10
+                    if element[Element.addressStart] < startPoint:
                         xAxeRectStart = 0
-                        rectLength = element[Element.AddressEnd] - startPoint
-                        originalStartAddress = element[Element.OriginalAddressStart]
+                        rectLength = element[Element.addressEnd] - startPoint
+                        originalStartAddress = element[Element.originalAddressStart]
                         fontColour = "red"
                     if index == 0:
-                        biggestEndAddress = element[Element.AddressEnd]
+                        biggestEndAddress = element[Element.addressEnd]
                     else:
-                        # check if the actual element is overlapped by the last element
-                        if element[Element.AddressStart] <= elementstoPlot[index-1][Element.AddressStart] or element[Element.AddressStart] < elementstoPlot[index-1][Element.AddressEnd]:
+                        # Check if the actual element is overlapped by the last element
+                        if element[Element.addressStart] <= elementsToPlot[index - 1][Element.addressStart] or element[Element.addressStart] < elementsToPlot[index - 1][Element.addressEnd]:
```

Comment:
This is more readable
```suggestion
                        startAddrPrevElement = elementsToPlot[index - 1][Element.addressStart]
                        endAddrPrevElement = elementsToPlot[index - 1][Element.addressEnd]
                        # If the actual element is overlapped by the last element go to next row
                        if element[Element.addressStart] <= startAddrPrevElement or element[Element.addressStart] < endAddrPrevElement:
```